app-backend/datamodel/src/main/scala/StacExport.scala

```diff
@@ -145,6 +147,7 @@ object StacExport {
         ExportStatus.NotExported,
         this.taskStatuses.map(_.toString),
         Some(annotationProjectId),
+        None,
```

Comment:
@jisantuc Should this also take an optional `exportAssetTypes` as was done for `CampaignExport` with a corresponding change to the route?

Reply:
I don't think it needs to -- `writeStacCatalog` right now creates an empty zip if the the export definition doesn't have a campaign id, so these are basically unprocessable anyway. I think if we want to start exporting individual projects we can revisit. And it would be really hard for a user outside of the team to run into that rough edge.

Reply:
https://github.com/raster-foundry/raster-foundry/blob/develop/app-backend/batch/src/main/scala/stacExport/WriteStacCatalog.scala#L286-L303